src/components/WebexActivityStream/WebexActivityStream.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import RoomsJSONAdapter from '../../adapters/RoomsJSONAdapter';
+import {RoomType} from '../../adapters/RoomsAdapter';
+import rooms from '../../data/rooms';
+
+import WebexActivityStream from './WebexActivityStream';
+
+// Setup for the stories
+const [roomID] = Object.keys(rooms);
+const stories = storiesOf('Webex Activity Stream', module);
+const newRooms = {};
+
+// Stories
+stories.add('empty group stream', () => <WebexActivityStream roomID={roomID} adapter={new RoomsJSONAdapter(rooms)} />);
+stories.add('empty direct stream', () => {
```

Comment:
let's call it `1:1` instead of direct. Most of our [examples on widgets](https://developer.webex.com/docs/widgets/space-widget) call it 1:1.

Internally in the code, `DIRECT` is fine to use since that's the "type"